roomee-api/roomee-api/Models/Event.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Security.Claims;
+
+namespace roomee_api.Models
+{
+    public class Event{
+        [JsonProperty(PropertyName = "eventId")]
+        public int EventId { get; }
+
+        [JsonProperty(PropertyName = "creatorUserId")]
```

Comment:
CreatorUserId and CreationTimestamp will not need to have a default set function as these are configured only by the API and not by the front-end. This is the same as EventId and StatusId because the front-end does not set these initially.